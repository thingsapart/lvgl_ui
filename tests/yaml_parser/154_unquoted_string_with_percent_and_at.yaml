# The '%' and '@' characters are allowed in unquoted strings.
# The parser should handle them correctly.
discount: 10%
user: @admin
message: Sale is 20% off for @everyone!
