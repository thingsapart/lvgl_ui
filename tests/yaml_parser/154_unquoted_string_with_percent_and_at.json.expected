[{
		"discount":	"10%",
		"user":	"@admin",
		"message":	"Sale is 20% off for @everyone!"
	}]
